pkg/controller/vsphere/actuator_test.go

```diff
@@ -48,6 +48,7 @@ func TestMachineEvents(t *testing.T) {
 	defer server.Close()
 
 	host, port, err := net.SplitHostPort(server.URL.Host)
+	fmt.Printf("Host: %v Port: %v\n", host, port)
```

Comment:
Debug?

Reply:
I can remove this. I like this output so when test failed, i can verify the port number that was being used matched.